helm/kanister-operator/values.yaml

```diff
@@ -22,6 +22,7 @@ controller:
   # false : CRDs would be created by helm
   # true : CRDs would be created by kanister controller
   updateCRDs: true
+  parallelism: 8
```

Comment:
Do you think it would make more sense if we introduce another field here called `repoServerController` and then add `parallelism` under that?

Reply:
@viveksinghggits, My thoughts were, that kopia commands that we construct are being constructed by the kanister-controller, so that's why I put parallelism under controller.
Let me know WDYT about this

Reply:
even though it's being constructed by kanister controller. Is this parallelism related to/is for kanister controller or repo server controller?

Reply:
It's for kanister controller !, only our BackupDataUsingKopiaServer would be using it